brush/src/components/Map/DrawingLayer/index.js

```diff
@@ -1,43 +1,46 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Graphics } from '@inlet/react-pixi'
-import { getPointRef } from '../globalState'
+import { fromSchemeGetTileNameById } from 'scissors'
 
 const DrawingLayer = ({
+  getTilemapPoint,
   height,
   resolution,
+  scheme,
+  setSelectedPointInfos,
   toolState,
+  updateTilemapLayer,
   updateTilemapPoint,
   width,
 }) => {
   const { name: toolName, value: toolValue } = toolState
 
   const mapToolToFunc = {
-    brush: ({
-      paddingY, ref, x, y,
-    }) => {
-      ref.paintTile(toolValue, paddingY)
+    brush: ({ x, y }) => {
       updateTilemapPoint(x, y, toolValue)
+      updateTilemapLayer()
     },
 
-    eraser: ({
-      paddingY, ref, x, y,
-    }) => {
-      ref.paintTile(0x00, paddingY)
+    eraser: ({ x, y }) => {
       updateTilemapPoint(x, y, toolValue)
+      updateTilemapLayer()
     },
 
-    magnifyingGlass: ({ paddingY, ref }) => {
-      ref.clickHandle(paddingY)
+    magnifyingGlass: ({ x, y }) => {
+      const tileId = getTilemapPoint(x, y)
+      const tilemapPointInfos = {
+        message: `Tile ${fromSchemeGetTileNameById(scheme)(tileId)} (${tileId})`,
```

Comment:
This message could be mounted in a function outside the component.
```suggestion
        message: mountMagnifyingMessage(scheme, tileId, tileId),
```